ci/tests/puppeteer/scenarios/pm-must-change-password/script.js

```diff
@@ -0,0 +1,31 @@
+const puppeteer = require('puppeteer');
```

Comment:
Please rename the scenario to indicate "-with-activedirectory" or something similar.

Reply:
I changed it to `-ldap-ad` but don't feel strongly one way or the other.